doc/source/technology_showcase_examples/techdemo-25/ex_25-tecstent.ipynb

```diff
@@ -1,514 +1 @@
,{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.nsol(2, monitored_node, \"u\", \"y\")"]},{"cell_type":"markdown","metadata":{},"source":["compute the response power spectral density for displacement associated with<br>\n","variable 2"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.rpsd(3, 2)\n","mapdl.show(\"png\")"]},{"cell_type":"markdown","metadata":{},"source":["plot the variable 3"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.plvar(3)"]},{"cell_type":"markdown","metadata":{},"source":["print the variable 3"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.prvar(3)"]},{"cell_type":"markdown","metadata":{},"source":["x-axis is set for Log X scale"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.gropt(\"logx\", 1)"]},{"cell_type":"markdown","metadata":{},"source":["y-axis is set for Log X scale"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.gropt(\"logy\", 1)"]},{"cell_type":"markdown","metadata":{},"source":["plot the variable 3"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.plvar(3)\n","mapdl.show(\"close\")"]},{"cell_type":"markdown","metadata":{},"source":["#############################################################################<br>\n","Post-process PSD analysis using Matplotlib<br>\n","~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"]},{"cell_type":"markdown","metadata":{},"source":["store MAPDL results to python variables"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.dim(\"frequencies\", \"array\", 4000, 1)\n","mapdl.dim(\"response\", \"array\", 4000, 1)\n","mapdl.vget(\"frequencies\", 1)\n","mapdl.vget(\"response\", 3)\n","frequencies = mapdl.parameters[\"frequencies\"]\n","response = mapdl.parameters[\"response\"]"]},{"cell_type":"markdown","metadata":{},"source":["use Matplotlib to create graph"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["fig = plt.figure()\n","ax = fig.add_subplot(111)\n","plt.xscale(\"log\")\n","plt.yscale(\"log\")\n","ax.plot(frequencies, response)\n","ax.set_xlabel(\"Frequencies\")\n","ax.set_ylabel(\"Response power spectral density\")"]},{"cell_type":"markdown","metadata":{},"source":["## Exit MAPDL"]},{"cell_type":"code","execution_count":null,"metadata":{},"outputs":[],"source":["mapdl.exit()"]}],"metadata":{"kernelspec":{"display_name":"Python 3.10.8 ('.venv': venv)","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.10.8"},"vscode":{"interpreter":{"hash":"25ee88bf99fdb66867962637ab0ae1b8de27a896c2eb41e2fbf5e9143b58a926"}}},"nbformat":4,"nbformat_minor":2}
```

Comment:
All in one line?

Reply:
I don't have this visual on VS, I'll have a closer look to solve this problem